pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -472,7 +474,7 @@ func updateCertTimer(configInterval uint32, tickerHandle interface{}) {
 // Start by trying the all the free management ports and then all the non-free
 // until one succeeds in communicating with the cloud.
 // We use the iteration argument to start at a different point each time.
-// Returns a configProcessingSkipFlag
+// Returns a configProcessingSkipFlagReboot
```

Comment:
Comment doesn't match return type.

Reply:
But that was also true before your changes.

Reply:
Fixed